testing/trino-testing-services/src/main/resources/junit-platform.properties

```diff
@@ -0,0 +1 @@
+junit.jupiter.extensions.autodetection.enabled=true
```

Comment:
This should go in the surefire configuration: https://junit.org/junit5/docs/current/user-guide/#running-tests-build-maven-config-params